checkQC/run_type_recognizer.py

```diff
@@ -29,6 +29,28 @@ def __init__(self, config, runfolder):
         with open(os.path.join(self._runfolder, "RunInfo.xml")) as f:
             self._run_info = xmltodict.parse(f.read())
 
+        with open(self._find_run_parameters_xml()) as f:
+            self._run_parameters = xmltodict.parse(f.read())
+
+    def _find_run_parameters_xml(self):
+        first_option = os.path.join(self._runfolder, "RunParameters.xml")
```

Comment:
Nice pattern. Will remember it for later, though for this case I find the code as is easier to understand.